src/style.css

```diff
@@ -0,0 +1,123 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("/fonts/Avenir.ttc"),
+    url("fonts/Avenir-Book.ttf");
+}
+@font-face {
+  font-family: "Avenir";
+  src: url("/fonts/Avenir.ttc"),
+    url("/fonts/Avenir-Heavy.ttf");
+}
```

Comment:
You should add correct font-weight for each font-face and use it for other elements

Reply:
Check my last message in main channel in slack